lib/cnf_graph.ml

```diff
@@ -49,6 +39,20 @@ let get_variable_value var dim =
   let x = x mod dim in
   (not, x, y, t, c)
 
+let pp_var dim fmt var =
```

Comment:
The problem is that the coef of the base is not the dimension of the grid. It depends on the large value between:
-The number of colours
-The width of the grid
-Height of grid
-Maximum time

Reply:
I don't understand : I use exactly the same variable `dim` you use to call `get_variable_value`, which is what I need
If It's wrong, all the functions that use `dim` are wrong too no ?

Reply:
yes they are wrong 😔... I edited this in branch 2-CNF

Reply:
ok it's now fixed